src/current/v23.1/security-reference/security-overview.md

```diff
@@ -165,11 +165,11 @@ CockroachDB {{ site.data.products.core }} here refers to the situation of a user
    <td>Network-level Configuration of allowed IP addresses</td>
  </tr>
  <tr>
-  <td>&nbsp;</td>
+  <td>✓<a href="#privatelink-multiregion-serverless"><sup>1</sup></a></td>
   <td>✓</td>
   <td>✓</td>
   <td>✓</td>
-  <td><a href="https://www.cockroachlabs.com/docs/cockroachcloud/create-your-cluster.html#step-8-enable-vpc-peering-optional">VPC Peering</a> for GCP clusters and <a href="https://www.cockroachlabs.com/docs/cockroachcloud/aws-privatelink">AWS PrivateLink</a> for AWS clusters </td>
+  <td><a href="https://www.cockroachlabs.com/docs/cockroachcloud/connect-to-your-cluster.html#private-service-connect">Private Service Connect (PSC) (Preview)</a> or <a href="https://www.cockroachlabs.com/docs/cockroachcloud/connect-to-your-cluster.html#vpc-peering">VPC Peering</a> for GCP clusters and <a href="https://www.cockroachlabs.com/docs/cockroachcloud/aws-privatelink">AWS PrivateLink</a> for AWS clusters </td>
```

Comment:
I believe this syntax can be `{% link cockroachcloud/connect-to-your-cluster.md %}#the-anchor` however writing that out is such a faff and doesn't save many characters anyway so I leave it to your conscience / perfectionism or lack thereof (or how much of a rush you're in)